app/code/Magento/Authorization/Test/Unit/Model/Acl/AclRetrieverTest.php

```diff
@@ -7,26 +7,36 @@
 namespace Magento\Authorization\Test\Unit\Model\Acl;
 
 use Magento\Authorization\Model\Acl\AclRetriever;
-
 use Magento\Authorization\Model\ResourceModel\Role\Collection as RoleCollection;
 use Magento\Authorization\Model\ResourceModel\Role\CollectionFactory as RoleCollectionFactory;
 use Magento\Authorization\Model\ResourceModel\Rules\Collection as RulesCollection;
 use Magento\Authorization\Model\ResourceModel\Rules\CollectionFactory as RulesCollectionFactory;
 use Magento\Authorization\Model\Role;
+use Magento\Authorization\Model\Rules;
 use Magento\Authorization\Model\UserContextInterface;
+use Magento\Framework\Acl;
+use Magento\Framework\Acl\Builder;
+use Magento\Framework\Exception\AuthorizationException;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
 
 /**
+ * @covers \Magento\Authorization\Model\Acl\AclRetriever
+ *
  * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
  */
-class AclRetrieverTest extends \PHPUnit\Framework\TestCase
+class AclRetrieverTest extends TestCase
 {
     /**
      * @var AclRetriever
      */
-    protected $aclRetriever;
+    private $aclRetriever;
 
-    /** @var \PHPUnit_Framework_MockObject_MockObject|Role $roleMock */
-    protected $roleMock;
+    /**
+     * @var Role|MockObject
+     */
+    private $roleMock;
 
     protected function setup()
```

Comment:
```suggestion
    protected function setUp(): void
```

Reply:
Hi @dmytro-ch, 
Thank you for your review. 
I have fixed all issues. 
Could you please see my PR once again. 
Thank you.